src/context/WalletProvider/MobileWallet/components/MobileSuccess.tsx

```diff
@@ -33,23 +31,11 @@ export const MobileSuccess = ({ location }: MobileSetupProps) => {
 
         if (mnemonic) {
           await wallet.loadDevice({ mnemonic, deviceId })
-          const { name, icon } = MobileConfig
           const walletLabel = vault?.label ?? 'label'
-          dispatch({
-            type: WalletActions.SET_WALLET,
-            payload: {
-              wallet,
-              name,
-              icon,
-              deviceId,
-              meta: { label: walletLabel },
-              connectedType: KeyManager.Mobile,
-            },
-          })
-          dispatch({ type: WalletActions.SET_IS_CONNECTED, payload: true })
```

Comment:
same suggestion re: commenting this

Reply:
Not valid anymore as discussed we prefer the quickfix regarding the complexity of `WalletProvider`, changed in [5eacb01](https://github.com/shapeshift/web/pull/7280/commits/5eacb015dbea0fa87977f66ebe16bb50b6338894)